ios-sdk.xcodeproj/project.pbxproj

```diff
@@ -779,7 +1084,6 @@
 		};
 		DB7A33F51C8F8E8B009CC743 /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = EEFF8429A821CE3BB0733D53 /* Pods-ios-sdkTests.debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Removed. It got "added back" because this was a new target that was created so configuration from the old test target didn't carry over